.travis.yml

```diff
@@ -28,6 +34,8 @@ before_script:
 
   - export NODE_PATH="$(pwd)/node_modules"
 
+  - if [[ "$IOJS_VERSION" != "" ]]; then wget https://iojs.org/dist/v${IOJS_VERSION}/iojs-v${IOJS_VERSION}-linux-x64.tar.xz; tar xvf iojs-v${IOJS_VERSION}-linux-x64.tar.xz; pwd; fi
```

Comment:
> and pwd seems useless
> and you need to export the NODE_BIN variable in this case though

This is because I'm not finished yet and I want to see where the stuff will be extracted to to set proper NODE_BIN